source/_components/plugwise.markdown

```diff
@@ -0,0 +1,164 @@
+---
+title: "Plugwise"
+description: "Plugwise Climate integration."
+logo: plugwise.png
+ha_category: Climate
+ha_iot_class: Local Polling
+ha_release: 0.98
+---
+
+This enables [Plugwise](https://plugwise.com) [Anna](https://www.plugwise.com/en_US/products/anna) thermostats to be integrated. This integration talks locally to your **Smile** interface, you will need its password and IP address.
+
+The password can be found on the bottom of your Smile, it should consist of 6 characters. To find your IP address use the Plugwise App: 
+ - Open the Plugwise App and choose the 'Settings'-icon (&#9776;) and choose 'HTML-interface'. 
+ - Go to the (lower) 'Settings'-icon (&#9776;) and choose 'Preferences'. 
+ - Choose 'System' then 'Networking' and your IP address will be shown.
+
+## Configuration
+
+You have to add the following to your configuration.yaml file.
+
+```yaml
+# Minimal configuration.yaml entry
+climate:
+  - platform: plugwise
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
```

Comment:
```suggestion
    host: YOUR_SMILE_LOCAL_IP
```

Placeholders need to be in capital letters, descriptions and instructions go into the configuration section, so please remove those comments.